gqlalchemy/models.py

```diff
@@ -100,7 +100,7 @@ class MemgraphKafkaStream(MemgraphStream):
     consumer_group: str = None
     batch_interval: str = None
     batch_size: str = None
-    bootstrap_servers: str = None
+    bootstrap_servers: Union[str, List] = None
```

Comment:
List of what?

Reply:
List of servers, as it is mentioned in the issue [https://github.com/memgraph/gqlalchemy/issues/93](url)